opentelemetry-sdk/src/metrics/aggregation.rs

```diff
@@ -62,6 +62,32 @@ pub enum Aggregation {
         /// instances.
         record_min_max: bool,
     },
+
+    /// An aggregation that summarizes a set of measurements as a histogram with
+    /// bucket widths that grow exponentially.
+    Base2ExponentialHistogram {
+        /// The maximum number of buckets to use for the histogram.
+        max_size: u32,
+
+        /// The maximum resolution scale to use for the histogram.
+        ///
+        /// The maximum value is `20`, in which case the maximum number of buckets
+        /// that can fit within the range of a signed 32-bit integer index could be
+        /// used.
+        ///
+        /// The minimum value is `-10` in which case only two buckets will be used.
+        max_scale: i8,
+
+        /// Indicates whether to not record the min and max of the distribution.
+        ///
+        /// By default, these values are recorded.
+        ///
+        /// It is generally not valuable to record min and max for cumulative data
+        /// as they will represent the entire life of the instrument instead of just
+        /// the current collection cycle, you can opt out by setting this value to
+        /// `false`
+        record_min_max: bool,
```

Comment:
There are default value for these parameters as per the specs - max_size : 160, max_scale: 20. Would it be possible for application to not provide these values, and use the default one ? If not, something we can support in subsequent PR.